tiledb/sm/misc/logger.cc

```diff
@@ -32,7 +32,11 @@
 
 #include "tiledb/sm/misc/logger.h"
 
+#ifdef _WIN32
 #include <spdlog/sinks/stdout_sinks.h>
+#else
+#include <spdlog/sinks/stdout_color_sinks.h>
```

Comment:
@cngzhnp Are you saying in your environment that `stdout_color_sinks.h` is available but `stdout_sinks.h` is not? CI has failed because `stdout_color_sinks.h` is not build in the superbuild currently.

Reply:
In standart output with no color, stdout_sinks in okay. However, if standart output with color needs another header file which I added in this change. 

I knew that it was failed. I am not sure that it is valid or not for CI. Please check this [link](https://github.com/gabime/spdlog/blob/be2a7515136b47d15aa21d93582ea6be97e7b533/example/example.cpp) at line 88-89.

Am I missing something in here?

Reply:
> However, if standart output with color needs another header file needed which I added in this change.
Can you clarify what you mean by this? What does 'with color' mean?

Reply:
I can't quite put my figure on what is happening either but it seems like there is difference between us pulling spdlog into `build/external`, or taking it from the system.  In the latter case I also had to mod one line every now and then for the another define (with color, if memory serves).

Reply:
@joe-maley `stdout_color_mt` function can not be found under this `stdout_sinks.h` header file. It should be located under `stdout_color_sinks.h` file IMHO. Maybe there is a difference between your spdlog version and mine. Please take a look [this version](https://github.com/gabime/spdlog/blob/v1.5.0/include/spdlog/sinks/stdout_sinks.h), function that I mentioned above is not defined under this header file. What do you think?

Reply:
I will remove that part from PR because of no-repro case and happens on only my environment. So, reverted back.